conductr_cli/conduct_main.py

```diff
@@ -328,6 +328,7 @@ def run(_args=[]):
     parser = build_parser(dcos_mode)
     argcomplete.autocomplete(parser)
     args = parser.parse_args(_args[1:])
+    args.dcos_mode = dcos_mode
```

Comment:
Unfortunately this is not possible because we need to use `dcos_mode` to build the `parser` in order for us to obtain the `args`:

https://github.com/typesafehub/conductr-cli/pull/181#discussion-diff-86314479R328